packages/env/src/index.ts

```diff
@@ -164,5 +164,5 @@ export function setupGlobal() {
   globalThis.$AFFINE_SETUP = true;
 }
 
-export { config };
+export { config, isElectron };
```

Comment:
can we add `isElectron` inside `getEnvironment`?

Reply:
yes，will fix

Reply:
remove isElectron and use isDesktop